Mage.Tests/src/test/java/org/mage/test/player/TestPlayer.java

```diff
@@ -2426,15 +2447,31 @@ public boolean chooseTarget(Outcome outcome, Target target, Ability source, Game
                     if (!targetDefinition.startsWith("targetPlayer=")) {
                         continue;
                     }
-                    checkTargetDefinitionMarksSupport(target, targetDefinition, "=");
-                    String playerName = targetDefinition.substring(targetDefinition.indexOf("targetPlayer=") + 13);
```

Comment:
1. Old code used in 3 places, but changed in one. Looks like it miss for `handlePlayerTarget`:
![shot_240521_170617](https://github.com/magefree/mage/assets/8344157/b53774f7-e8df-4dc0-ab3c-a4ed3b3ec610)

2. I recommends to use real targets lifecycle in tests by `addTarget` commands, not multi-targets by `^` (the last one added it directly and can be potentially broken in dynamic targets):
![shot_240521_171301](https://github.com/magefree/mage/assets/8344157/e325e4a3-2511-42a2-b5b1-b72006d2404a)